index.js

```diff
@@ -154,8 +179,7 @@ function encode(input) {
     else if (input.startsWith('%')) return encoder.message(input)
     else if (input.startsWith('&')) return encoder.blob(input)
     else if (input.endsWith('.sig.ed25519')) return encoder.signature(input)
-    else if (input.endsWith('.box2') || input.endsWith('.box'))
-      return encoder.box(input)
+    else if (input.match(/\.box\d*$/)) return encoder.box(input)
```

Comment:
Generalised this case to let e.g. `asdsadsa=.box40` through. in `encoder.box` this is then not matched by anything, leading to a throw

Reply:
Wait, do you want this code to throw on `.box` or not? Currently it wouldn't, because the regex is `/\.box\d*$/` not `/\.box\d+$/`.

Note, we have strings suffixed with `.box` in the wild. In fact I think ssb-bfe used to handle `.box1` but this was buggy with test input data, it had to be `.box`.

Reply:
I don't want it throwing on .box
I guess .box === .box1,  we should clarify in the encoder